docs/install_guides/installing-model-dev-tools.md

```diff
@@ -91,65 +91,67 @@ Note that the commands are different for a Python installation and a C++ install
+   To install and configure the components of the development package for working with specific frameworks, use the following command:
 
    .. code-block:: sh
-   
+
       pip install openvino-dev[extras]
-      
-  where the `extras` parameter specifies one or more deep learning frameworks via these values: `caffe`, `kaldi`, `mxnet`, `onnx`, `pytorch`, `tensorflow`, `tensorflow2`. Make sure that you install the corresponding frameworks for your models.
-  
-  For example, to install and configure the components for working with TensorFlow 2.x and ONNX, use the following command:
-  
+
+   where the `extras` parameter specifies one or more deep learning frameworks via these values: `caffe`, `kaldi`, `mxnet`, `onnx`, `pytorch`, `tensorflow`, `tensorflow2`. Make sure that you install the corresponding frameworks for your models.
+
+   For example, to install and configure the components for working with TensorFlow 2.x and ONNX, use the following command:
+
    .. code-block:: sh
-   
+
       pip install openvino-dev[tensorflow2,onnx]
-   
+
    .. note::
-      
+
       Model Optimizer support for TensorFlow 1.x environment has been deprecated. Use the `tensorflow2` parameter to install a TensorFlow 2.x environment that can convert both TensorFlow 1.x and 2.x models. If your model isn't compatible with the TensorFlow 2.x environment, use the `tensorflow` parameter to install the TensorFlow 1.x environment. The TF 1.x environment is provided only for legacy compatibility reasons.
-      
+
 .. tab:: C++
 
-  When using OpenVINO Development Tools for C++ development, it’s important to install the same version as OpenVINO Runtime. Following the instructions below will ensure that you are installing a version that matches that of OpenVINO Runtime.
-  
-  **Recommended: Install Using the Requirements Files**
-  After you have installed OpenVINO Runtime from an archive file, you can find a set of requirements files in the <INSTALL_DIR>\tools\ directory. The requirements files will install the matching version of OpenVINO Development Tools and its dependencies.
-  
-  1. Install the OpenVINO Development Tools mandatory requirements using the following command:
+   When using OpenVINO Development Tools for C++ development, it’s important to install the same version as OpenVINO Runtime. Following the instructions below will ensure that you are installing a version that matches that of OpenVINO Runtime.
+
+   **Recommended: Install Using the Requirements Files**
+
+   After you have installed OpenVINO Runtime from an archive file, you can find a set of requirements files in the <INSTALL_DIR>\tools\ directory. The requirements files will install the matching version of OpenVINO Development Tools and its dependencies.
+
+   1. Install the OpenVINO Development Tools mandatory requirements using the following command:
+      .. code-block:: sh
```

Comment:
@sgolebiewski-intel Unfortunately the formatting here is still not correct. :(

Reply:
Have corrected with the following commit: https://github.com/openvinotoolkit/openvino/pull/14492/commits/bf2e25af8fcf9faa12e6488ecfa5771de6cebe9d